qa/rpc-tests/mempool_limit.py

```diff
@@ -38,7 +38,6 @@ def run_test(self):
         self.nodes[0].settxfee(0) # return to automatic fee selection
         txFS = self.nodes[0].signrawtransaction(txF['hex'])
         txid = self.nodes[0].sendrawtransaction(txFS['hex'])
-        self.nodes[0].lockunspent(True, [us0])
```

Comment:
@jonasschnelli I fail to see why you added this noop. I guess you used it for debugging and it is save to remove.